### DISCUSSION TITLE
Deploy nextjs on normal web hosting

### DISCUSSION THREAD

**Discussion by TOcvfan** [vercel/next.js#56510] [Help]
### Summary

I ran the build command without any problems, but unlike react, there is no explanation on how to deploy it..
I'm not using it as server I already have a restAPI for that, so I'm only using it for client side, but it's useless if I can't deploy it to my website.
so how do I deploy it to my webhotel?

**Comment by icyJoseph** [comment]
You need to create a Static HTML output. You do that with export mode. 

- https://nextjs.org/docs/app/building-your-application/deploying/static-exports

If you've done that already, then you have a `.out` folder, unless you changed it. Then you can just copy the contents of the `.out` folder to your hosting provider.

For routing you might have a bit of an issue, because Next.js outputs `.html` files for example `about.html`, so when you do SPA transitions from `/` to `/about`, it will work fine, but reloading in `/about`, __might__ show 404. If it does, then you need to config your hosting provider to route `/about` into `/about.html`: https://nextjs.org/docs/app/building-your-application/deploying/static-exports#deploying
